.github/workflows/trigger-regression-tests.yml

```diff
@@ -0,0 +1,112 @@
+name: Trigger Regression Tests
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Select the branch to run tests on'
+        required: true
+        default: 'main'
+  pull_request_target:
+    types: [labeled]
+
+env:
+  RUN_ID: ${{ github.run_id }}-${{ github.run_attempt }}
+
+jobs:
+  run:
+    if: >
+      (github.event_name == 'workflow_dispatch') ||
+      (github.event_name == 'pull_request_target' && github.event.label.name == 'regression-tests')
```

Comment:
Please add "regression-tests" label to file: `.github/labels.yml`

Reply:
added